### PR TITLE
Docs: updated catalog tables and views

### PR DESCRIPTION
**Pull request by mperezfuster**
Updating some GPDB 7 catalog tables and views.
Review pages:
https://docs-staging.vmware.com/en/draft/VMware-Tanzu-Greenplum/mireia-catalog/greenplum-database/ref_guide-system_catalogs-catalog_ref-tables.html
https://docs-staging.vmware.com/en/draft/VMware-Tanzu-Greenplum/mireia-catalog/greenplum-database/ref_guide-system_catalogs-catalog_ref-views.html

### CODE REVIEW

**Review by dyozie — APPROVED** [review]

File: gpdb-doc/markdown/ref_guide/system_catalogs/catalog_ref-views.html.md [hunk 1]
```diff
@@ -1172,6 +1172,25 @@ Cursors exist only for the duration of the transaction that defines them, unless
...
+## <a id="pg_exttable"></a>pg_exttable
+
+The `pg_exttable` system catalog viewis used to track external tables and web tables created by the `CREATE EXTERNAL TABLE` command.
+
+|column|type|references|description|
+|------|----|----------|-----------|
+|`reloid`|oid|pg\_class.oid|The OID of this external table.|
+|`urilocation`|text\[\]| |The URI location\(s\) of the external table files.|
+|`execlocation`|text\[\]| |The ON segment locations defined for the external table.|
+|`fmttype`|char| |Format of the external table files: `t` for text, or `c` for csv.|
```
Comment (soumyadeep2007) [inline gpdb-doc/markdown/ref_guide/system_catalogs/catalog_ref-views.html.md:1184]: Please add: `b` for custom

File: gpdb-doc/markdown/ref_guide/system_catalogs/catalog_ref-views.html.md [hunk 2]
```diff
@@ -1172,6 +1172,25 @@ Cursors exist only for the duration of the transaction that defines them, unless
...
+|`fmtopts`|text| |Formatting options of the external table files, such as the field delimiter, null string, escape character, etc.|
+|`options`|text\[\]| |The options defined for the external table.|
+|`command`|text| |The OS command to run when the external table is accessed.|
+|`rejectlimit`|integer| |The per segment reject limit for rows with errors, after which the load will fail.|
+|`rejectlimittype`|char| |Type of reject limit threshold: `r` for number of rows, or `p` for percent.|
+|`logerrors`|bool| |`1` to log errors, `0` to not.|
+|`encoding`|text| |The client encoding.|
+|`writable`|boolean| |`0` for readable external tables, `1` for writable external tables.|
```
Comment (soumyadeep2007) [inline gpdb-doc/markdown/ref_guide/system_catalogs/catalog_ref-views.html.md:1192]: boolean fields don't display `1` or `0`, they display `true` or `false`.

**Review by soumyadeep2007 — APPROVED** [review]
LGTM!

File: gpdb-doc/markdown/ref_guide/system_catalogs/catalog_ref-views.html.md [hunk 3]
```diff
@@ -1172,6 +1172,25 @@ Cursors exist only for the duration of the transaction that defines them, unless
 |`creation_time`|timestamptz| |The time at which the cursor was declared.|
 |`is_parallel`|boolean| |`true` if the cursor was declared `PARALLEL RETRIEVE`; `false` otherwise.|
 
+## <a id="pg_exttable"></a>pg_exttable
+
+The `pg_exttable` system catalog viewis used to track external tables and web tables created by the `CREATE EXTERNAL TABLE` command.
```
Comment (lisakowen) [inline gpdb-doc/markdown/ref_guide/system_catalogs/catalog_ref-views.html.md:1177]: viewis -> view is

File: gpdb-doc/markdown/ref_guide/system_catalogs/pg_am.html.md [hunk 4]
```diff
@@ -4,37 +4,10 @@ The `pg_am` table stores information about index access methods. There is one ro
...
-|`ambuildempty`|regproc|pg\_proc.oid|"Build empty index" function|
-|`ambulkdelete`|regproc|pg\_proc.oid|Bulk-delete function|
-|`amvacuumcleanup`|regproc|pg\_proc.oid|Post-`VACUUM` cleanup function|
-|`amcanreturn`|regproc|pg\_proc.oid|Function to check whether index supports index-only scans, or zero if none|
-|`amcostestimate`|regproc|pg\_proc.oid|Function to estimate cost of an index scan|
-|`amoptions`|regproc|pg\_proc.oid|Function to parse and validate `reloptions` for an index|
+|`oid`|oid| |Row identifier \(hidden attribute; must be explicitly selected\).|
+|`amname`|name| |Name of the access method.|
+|`amhandler`|regproc| | OID of a handler function responsible for supplying information about the access method.|
+|`amtype`|char| |`t` for table(including materialized views), `i` for index.|
```
Comment (lisakowen) [inline gpdb-doc/markdown/ref_guide/system_catalogs/pg_am.html.md:10]: table(inc -> table (inc

File: gpdb-doc/markdown/ref_guide/system_catalogs/pg_authid.html.md [hunk 5]
```diff
@@ -8,15 +8,15 @@ Because user identities are system-wide, `pg_authid` is shared across all databa
...
+|`oid`|oid| |Row identifier|
 |`rolname`|name| |Role name|
 |`rolsuper`|boolean| |Role has superuser privileges|
 |`rolinherit`|boolean| |Role automatically inherits privileges of roles it is a member of|
 |`rolcreaterole`|boolean| |Role may create more roles|
 |`rolcreatedb`|boolean| |Role may create databases|
-|`rolcatupdate`|boolean| |Role may update system catalogs directly. \(Even a superuser may not do this unless this column is true\)|
 |`rolcanlogin`|boolean| |Role may log in. That is, this role can be given as the initial session authorization identifier|
 |`rolreplication`|boolean| |Role is a replication role. That is, this role can initiate streaming replication and set/unset the system backup mode using `pg_start_backup` and `pg_stop_backup`.|
+|`rolbypassrls`|boolean| |Roles bypasses every row-level security policy.|
```
Comment (lisakowen) [inline gpdb-doc/markdown/ref_guide/system_catalogs/pg_authid.html.md:19]: general comment - most of the single sentence descriptions do not end with a period, not sure if we want to make this consistent.

**Review by lisakowen — APPROVED** [review]
just some minor edits requested

File: gpdb-doc/markdown/ref_guide/system_catalogs/pg_authid.html.md (hunk 5 above)
Comment (mperezfuster) [reply gpdb-doc/markdown/ref_guide/system_catalogs/pg_authid.html.md:19]: Thanks, I am never too sure and usually do what I see already been written. Will remove these.

**Pull request merged**